plugins/kernel_v2/config/vm.rb

```diff
@@ -467,6 +469,7 @@ def finalize!
         @box_download_location_trusted = false if @box_download_location_trusted == UNSET_VALUE
         @box_url = nil if @box_url == UNSET_VALUE
         @box_version = nil if @box_version == UNSET_VALUE
+        @box_download_options = {} if @box_download_options == UNSET_VALUE
```

Comment:
It would be good to have some validation of this option in the `validate` step! Then we can give some good error messages if this option is incorrectly set to the wrong type, or whatever users might end up doing.